browser-test/src/support/admin_programs.ts

```diff
@@ -125,7 +125,27 @@ export class AdminPrograms {
 
     await this.page.click('#add-block-button');
 
-    await this.page.type('textarea', blockDescription);
+    await this.page.fill('textarea', blockDescription);
+    await this.page.click('#update-block-button');
+
+    for (const questionName of questionNames) {
+      await this.page.click(`button:text("${questionName}")`);
+    }
+    return await this.page.$eval('#block-name-input', el => (el as HTMLInputElement).value);

```

Comment:
Suggestion (non-blocking): `await this.page.textContent('#block-name-input')` may be more readable.

Reply:
This wasn't working 🤷 . I can try it again.

Reply:
It don't work 😦 . I'm gonna keep what I have.